x-pack/plugins/lens/public/shared_components/axis/extent/axis_extent_settings.tsx

```diff
@@ -186,27 +185,27 @@ function MetricAxisBoundsControl({
               label: i18n.translate('xpack.lens.axisExtent.full', {
                 defaultMessage: 'Full',
               }),
-              'data-test-subj': `${testSubjPrefix}_axisExtent_groups_full'`,
+              'data-test-subj': `${testSubjPrefix}_axisExtent_groups_full`,
             },
             {
               id: `${idPrefix}dataBounds`,
               label: i18n.translate('xpack.lens.axisExtent.axisExtent.dataBounds', {
                 defaultMessage: 'Data',
               }),
-              'data-test-subj': `${testSubjPrefix}_axisExtent_groups_DataBounds'`,
-              isDisabled: shouldIncludeZero,
+              'data-test-subj': `${testSubjPrefix}_axisExtent_groups_data`,
+              isDisabled: hasBarOrArea,
```

Comment:
This is what we could add here:
```suggestion
              isDisabled: hasBarOrArea,
              toolTipContent: i18n.translate('xpack.lens.axisExtent.disabledDataBoundsMessage', {
                defaultMessage: 'Only line charts can be fit to the data bounds',
              }),
```

Reply:
Updated in 8c7a714.